lib/presentation/model/presentation_search.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:equatable/equatable.dart';
+import 'package:matrix/matrix.dart';
+
+enum SearchTypeEnum {
+  contact,
+  recentChat,
+}
+
+class PresentationSearch extends Equatable {
```

Comment:
`presentation contact` belong to `domain`, why this `presentation search` belong to `presentation`